### PR TITLE
feat: Adds a method.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

File: src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java [hunk 1]
```diff
@@ -374,6 +374,29 @@ public boolean matches(long ssrc)
     }
 
     /**
+     * Gets the maximum subjective quality index that complies to the max height
+     * specified as an argument.
+     *
+     * @param maxHeight the max height
+     * @return the maximum subjective quality index that complies to the max height
+     * specified as an argument, or 0.
```
Comment (bbaldino) [inline src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java]: 0 is a valid index though, isn't it? shouldn't it return -1 if it can't find anything?

Comment (gpolitis) [inline src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java]: Thanks @bbaldino, it's fixed.

**Comment by bbaldino** [comment]
cool.  lgtm.  ready to merge?

**Comment by gpolitis** [comment]
Yes, ready to merge. Let's just be courteous to Jenkins and wait for the tests to pass :)

**Comment by saghul** [comment]
Please reword the commit message adding what method was added and why.

Signed, the Git Hygiene police 👮 🚓

**Comment by gpolitis** [comment]
I've also squashed the commits (since the Git police is on patrol today !)

**Comment by saghul** [comment]
Thanks for complying with The Law. 🙇

**Pull request merged**